controllers/databasecluster_controller.go

```diff
@@ -106,10 +106,6 @@ wsrep_trx_fragment_size=3670016
 `
 	pxcMinimalConfigurationTemplate = `[mysqld]
 wsrep_provider_options="gcache.size=%s"
-`
-	haProxyDefaultConfigurationTemplate = `timeout client 28800s
-timeout connect 100500
-timeout server 28800s
```

Comment:
A bit concerned we just delete the default configuration. If the format is incorrect as the ticket says, is it possible to make it correct?

Reply:
The format is incorrect and we need to specify the whole HAProxy config (see the attached link to the documentation in the description of PR). Also, since we're changing timeouts it makes no sense because it's defaults that are set in the PXC operator (see the second link to the actual change in the PXC operator)

Reply:
I believe we don't need it anymore according to the description, nevernid